internal/nodes/crd_translator.go

```diff
@@ -107,6 +109,17 @@ func BuildL7HostRule(host, key string, vsNode AviVsEvhSniModel) {
 			Value:  hostrule.Namespace + "/" + hostrule.Name,
 			Status: lib.CRDActive,
 		}
+
+		if (akov1alpha1.HostRuleAnalyticsPolicy{}) != hostrule.Spec.VirtualHost.AnalyticsPolicy {
```

Comment:
@chauhanshubham Can you check if this is fine in case the field AnalyticsPolicy is omitted from the hostrule ? I think you had to use a pointer here ?

Reply:
I think this should be fine. Although I would prefer declaring `AnalyticsPolicy` to be of type `* HostRuleAnalyticsPolicy` and comparing here with nil value, but i have no good reason to use one or the other.
@swathinsankaran can you please check this scenario once to confirm.

Reply:
Yes, the pointer will be the best approach. I have changed the code accordingly.